common/laws/00_monarchy.txt

```diff
@@ -93,13 +93,13 @@ monarchy_military_reforms = {
 	
 	mercenary_reliance = {
 		modifier = {
-			mercenary_land_maintenance_cost = -0.20
+			mercenary_land_maintenance_cost = -0.25
 			recruit_mercenary_cost_modifier = -0.35
 			max_mercenary_stacks = 1
-			levy_size_multiplier = -0.05
+			levy_size_multiplier = -0.025
 			cohort_reinforcement_speed = 0.1
 			land_morale_recovery = 0.1
-			monthly_military_experience = 0.075
+			monthly_military_experience = 0.15
 			army_maintenance_cost = 0.5
```

Comment:
note that this is intended as a soft cap for nations that already have legions that wish to use lesser laws.